### PR TITLE
content updates from Steve's review

### PR DESCRIPTION
**Pull request by heckj**
Making updates based on steve's review - PR merged prior to his full review, so created another PR with the relevant changes

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7217)

### CODE REVIEW

**Comment by heckj** [comment]
@steveperry-53 I put a `hold` on this one so that it'll stay until you provide the thumb's up. I tried to address all the comments you made on the previous PR. To answer the prior question on Katacoda and Minikube, yes - I'm quite sure it uses minikube based on my interactions with updates to the "remote tutortials"

File: docs/user-journeys/users/cluster-operator/foundational.md [hunk 1]
```diff
@@ -27,19 +27,22 @@ Metrics
 
 If you have not already done so, start your understanding by reading through [What is Kubernetes?](/docs/concepts/overview/what-is-kubernetes/), which introduces a number of basic concepts and terms.
 
-Kubernetes is quite flexible, and can a cluster can be run in a wide variety of places. You can interact with Kubernetes entirely on your own laptop or local development machine with it running within a virtual machine. Kubernetes can also run on virtual machines hosted either locally or in a cloud provider, and you can run a kubernetes cluster on bare metal.
+Kubernetes is quite flexible, and a cluster can be run in a wide variety of places. You can interact with Kubernetes entirely on your own laptop or local development machine with it running within a virtual machine. Kubernetes can also run on virtual machines hosted either locally or in a cloud provider, and you can run a kubernetes cluster on bare metal.
```
Comment (Bradamant3) [inline docs/user-journeys/users/cluster-operator/foundational.md]: You've still got a stray lower case "k" in the second sentence here :-(

Comment (heckj) [reply docs/user-journeys/users/cluster-operator/foundational.md]: Oh bugger - thank you! On it!

File: docs/user-journeys/users/cluster-operator/foundational.md [hunk 2]
```diff
@@ -27,19 +27,22 @@ Metrics
...
+Katacoda provides a browser-based connection to a single-node cluster, using minikube behind the scenes, to support a number of tutorials to explore Kubernetes. You can also leverage the web-based [Play with Kubernetes](http://labs.play-with-k8s.com/) to the same ends - a temporary cluster to play with on the web.
 
 You interact with Kubernetes either through a dashboard, an API, or using a command-line tool (such as `kubectl`) that interacts with the Kubernetes API.
-Be familiar with how to [Organizing Cluster Access](/docs/concepts/configuration/organize-cluster-access-kubeconfig/) with configuration files.
+Be familiar with [Organizing Cluster Access](/docs/concepts/configuration/organize-cluster-access-kubeconfig/) by using configuration files.
 The Kubernetes API exposes a number of resources that provide the building blocks and abstractions that are used to run software on Kubernetes.
 Learn more about these resources at [Understanding Kubernetes Objects](/docs/concepts/overview/kubernetes-objects)
 These resources are covered in a number of articles within the Kubernetes documentation.
```
Comment (steveperry-53) [inline docs/user-journeys/users/cluster-operator/foundational.md]: Need a period after Understanding Kubernetes Objects.

File: docs/user-journeys/users/cluster-operator/intermediate.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ If you are a cluster operator looking to expand your grasp of Kubernetes, this p
 
 ## Work with Ingress, Networking, Storage, Workloads
```
Comment (steveperry-53) [inline docs/user-journeys/users/cluster-operator/intermediate.md]: All the other heading use sentence case, not title case. So I think this heading should be:

Work with ingress, networking, storage, workloads

**Comment by steveperry-53** [comment]
@heckj This looks good. Just two remaining comments in the text.

**Comment by heckj** [comment]
Thanks @steveperry-53 - updated!

**Comment by heckj** [comment]
@steveperry-53 toss in a `lgtm` if you think it’s ready to merge, I’ll remove the hold here
/hold cancel

**Pull request merged**